src/photos.cs

```diff
@@ -1005,9 +1005,15 @@ interface PHPhotoLibrary {
 
 	[iOS (9,1)]
 	[TV (10,0)]
+#if MONOMAC
+	[DisableDefaultCtor] // NS_UNAVAILABLE
```

Comment:
good then :)

btw there are introspection-mac test errors reported - but they don't look related to this change?!?

Unified (Failed) Show details
Unified XM45 (Failed) Show details